doc/source/howtos/upgrade_guide.rst

```diff
@@ -146,60 +120,38 @@ To upgrade the cluster, follow these steps for each node:
 #. Back up :file:`grastate.dat`, so that you can restore it
    if it is corrupted or zeroed out due to network issue.
 
-#. Start the node outside the cluster (in standalone mode)
-   by setting the :variable:`wsrep_provider` variable to ``none``.
-   For example:
-
-   .. code-block:: bash
-
-      $ sudo mysqld --skip-grant-tables --user=mysql --wsrep-provider='none'
-
-   .. note::
-
-      To prevent any users from accessing this node while performing
-      work on it, you may add `--skip-networking
-      <https://dev.mysql.com/doc/refman/8.0/en/server-options.html#option_mysqld_skip-networking>`_
-      to the startup options and use a local socket to connect, or
-      alternatively you may want to divert any incoming traffic from
-      your application to other operational nodes.
-
-#. When the upgrade is done, stop the ``mysqld`` process.
-   You can either run ``sudo kill`` on the ``mysqld`` process ID,
-   or ``sudo mysqladmin shutdown`` with the MySQL root user credentials.
-
-   .. note:: On CentOS, the :file:`my.cnf` configuration file
-      is renamed to :file:`my.cnf.rpmsave`.
-      Make sure to rename it back
-      before joining the upgraded node back to the cluster.
 
 #. Now you can join the upgraded node back to the cluster.
 
-   In most cases, starting the ``mysql`` service
-   should run the node with your previous configuration::
+   In most cases, starting the ``mysql`` service should run the node with your
+   previous configuration. For more information, see :ref:`add-node`.
```

Comment:
This doesn't mention `pxc-encrypt-cluster-traffic` on 8.0 it's enabled by default. In case you don't configure it and/or try to join 5.7 cluster which is running unencrypted it won't be able to join and result in:
```--------
2020-04-28T18:18:22.114709Z 0 [ERROR] [MY-000000] [Galera] handshake with remote endpoint ssl://192.168.70.63:4567 failed: asio.ssl:336130315: 'wrong version number' ( 336130315: 'error:1408F10B:SSL routines:ssl3_get_record:wrong version number')
This error is often caused by SSL issues. For more information, please see:
  https://per.co.na/pxc/encrypt_cluster_traffic
--------
2020-04-28T18:18:23.115089Z 0 [ERROR] [MY-000000] [Galera] handshake with remote endpoint ssl://192.168.70.61:4567 failed: asio.ssl:336130315: 'wrong version number' ( 336130315: 'error:1408F10B:SSL routines:ssl3_get_record:wrong version number')
This error is often caused by SSL issues. For more information, please see:
  https://per.co.na/pxc/encrypt_cluster_traffic```